### ISSUE TITLE
Rescanning "all your notes"

### ISSUE THREAD

**Issue by feedhopper** [Sergei-Korneev/obsidian-local-images-plus#7]
The approach of the plugin is fantastic. I loved the old one and from what I see, your new version is going to improve it a lot. Thanks again for that!

One thing I always got annoyed by: I use a lot the option to "Download images locally for all your notes".
There are two griefs I have with that:

- It takes AGES with my vault as I have a lot of content with images and schematics.
- Some articles "hang" for a long time, I can see them stalling in the little progress window.

Now I wonder:
Is there any way to exclude notes that have not changed since the last download?
This would save so much time.

All the best!

**Comment by Sergei-Korneev** [comment]
Yes , I thought about that too. Note that I would  recommend to use command for all notes with caution on 0.14.5, since some parsing issues were found after release, and some notes may lost some images.

In version 0.14.6 I tried to improve parsing  as much as I can.
